src/targets/shell/curl.js

```diff
@@ -44,10 +44,11 @@ module.exports = function (source, options) {
   switch (source.postData.mimeType) {
     case 'multipart/form-data':
       source.postData.params.map(function (param) {
-        var post = util.format('%s=%s', param.name, param.value)
-
-        if (param.fileName && !param.value) {
+        var post = ''
+        if (param.fileName) {
```

Comment:
Was `!param.value` intentionally removed from this condition? Just comparing with the previous implementation. 😄

Reply:
Making this here and in the node `request` target because if `param.fileName` is present, those targets have the ability to handle uploading the file via the path instead of dealing with the contents of the file that may or may not be included in `param.value`.